packages/n4s/src/rules/__tests__/isKeyOf.test.ts

```diff
@@ -0,0 +1,36 @@
+import { isKeyOf } from 'isKeyOf';
```

Comment:
These tests are great, but can you also add tests for the function when used within enforce?

```js
it('Should pass when ...', () => {
  enforce(...).isKeyOf({...}) // passing values
})
```

```js
it('Should fail when ...', () => {
  expect(() => enforce(...).isKeyOf({...})).toThrow() // failing values
})
```